File: generated-sources/php-ze-ph/mojang-api/src/App/com/github/asyncmc/mojang/api/php/ze/ph/model/ChangeSkinRequest.php
<?php

namespace com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Request Mojang to download a skin from an URL and apply to the player
 */
class ChangeSkinRequest
{
    /**
     * @DTA\Data(field="model", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model\SkinModel::class})
     * @DTA\Validator(name="Dictionary", options={"type":\com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model\SkinModel::class})
     * @var \com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model\SkinModel
     */
    public $model;
    /**
     * The URL which Mojang servers will download and apply the skin
     * @DTA\Data(field="url")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $url;
}

File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/com/github/asyncmc/mojang/api/scala/gatling/model/ChangeSkinRequest.scala

package com.github.asyncmc.mojang.api.scala.gatling.model


case class ChangeSkinRequest (
    _model: Option[SkinModel],
    /* The URL which Mojang servers will download and apply the skin */
    _url: String
)
object ChangeSkinRequest {
    def toStringBody(var_model: Object, var_url: Object) =
        s"""
        | {
        | "model":$var_model,"url":$var_url
        | }
        """.stripMargin
}
